testing/helpers/helpers.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+// AssertDosFile parses the file and ensures that every line ending is formatted
+// for DOS operating systems with a carriage return and line feed ("\r\n").
+func AssertDosFile(t *testing.T, file []byte) {
+	fixture := string(file)
+	for i := range fixture {
+		if i == 0 {
+			continue
+		}
+		if fixture[i] == '\n' {
+			assert.Equal(t, uint8('\r'), fixture[i-1])
+		}
+	}
+}
+
+// AssertUnixFile parses the file and ensures that every line ending is formatted
+// for Unix/Linux operating systems with only line feed ("\n").
+func AssertUnixFile(t *testing.T, file []byte) {
+	fixture := string(file)
+	for i := range fixture {
+		if i == 0 {
+			continue
+		}
+		if fixture[i] == '\n' {
+			assert.NotEqual(t, uint8('\r'), fixture[i-1])
+		}
+	}
+}
+
+// PackageInTransitiveGraph searches a map (typically from Graph.Deps.Transitive)
+// for a package and returns it if it exists.
+func PackageInTransitiveGraph(packages map[pkg.ID]pkg.Package, name, revision string) pkg.Package {
+	for id := range packages {
```

Comment:
Go needs to add methods such as `.Contains`, `.Find`, `.Map`, etc to slices and arrays... 🙄  Alas that's probably never going to happen

Reply:
to the `map` type too

Reply:
Yea, I think once generics land there will be more libraries for this.